packages/runtime/test/consumption/attributes.test.ts

```diff
@@ -2342,3 +2329,116 @@ describe("DeleteAttributeUseCases", () => {
         });
     });
 });
+
+describe("Third party relationship attributes", () => {
+    let localAttribute: LocalAttributeDTO;
+    beforeAll(async () => {
+        await cleanupAttributes();
+        localAttribute = await executeFullCreateAndShareRelationshipAttributeFlow(services1, services2, {
+            content: {
+                key: "ThirdPartyKey",
+                confidentiality: RelationshipAttributeConfidentiality.Public,
+                value: {
+                    "@type": "ProprietaryString",
+                    value: "ThirdPartyValue",
+                    title: "ThirdPartyTitle"
+                },
+                isTechnical: true
+            }
+        });
+    });
+
+    test("Should share an third party attribute that i created", async () => {
```

Comment:
Regarding the names of the following test:
- usually we start with lowercase letters
- please write "ThirdPartyRelationshipAttribute" instead of "third party attribute" for consistency
- I've never seen "I" or "me" seen in our tests, so please adjust those names accordingly.

Reply:
Make sure to mind the first two points also in the other tests.